arcgis-ios-sdk-samples/Display information/Show legend/MILLegendTableViewController.swift

```diff
@@ -28,7 +28,7 @@ class MILLegendTableViewController: UITableViewController {
         self.populateLegends(self.operationalLayers as AnyObject as! [AGSLayerContent])
     }
     
-    func populateLegends(layers:[AGSLayerContent]) {
+    func populateLegends(_ layers:[AGSLayerContent]) {
```

Comment:
for this I did `func populateLegends(with layers:[AGSLayerContent])`. In this case its easy to use a proposition because the attribute type is meaningful or specific. So when someone is using the method, they get `self.populateLegends(with: <#T##[AGSLayerContent]#>)` which is self explanatory.